src/admin/BulkUserCreationPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react';
+import { Box, Label, Heading, Container, Button, Text, Textarea, Alert, Select } from 'theme-ui';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import { Role, CreateUserCommand } from '../api';
+import useBulkUserCreation from './hooks/useBulkUserCreation';
+
+const AnyBox = Box as any;
+
+const ROLES: Role[] = ['TRUSTED_PATIENT', 'CLINICIAN'];
+
+const BulkUserCreationPage: FC = () => {
+  const { create, loading, error, createdUsers } = useBulkUserCreation();
+
+  const form = useFormik<{ emailsString: string; role: Role }>({
+    initialValues: {
+      emailsString: '',
+      role: 'TRUSTED_PATIENT',
+    },
+    validationSchema: yup.object().shape({
+      emailsString: yup.string().required(),
+      role: yup.string().required(),
+    }),
+    onSubmit: ({ emailsString, role }) => {
+      const emails = getEmails(emailsString);
+      const command: CreateUserCommand[] = emails.map((email) => ({ email, roles: [role] }));
+      create(command);
+    },
+  });
+
+  return (
+    <Container variant="page">
+      <Heading as="h1" mb={4}>
+        Create users
+      </Heading>
+
+      <Text mb={4}>Users with the emails will be created with the selected role.</Text>
```

Comment:
Any copy improvement ideas?

Reply:
Not sure if it's better. 
```
Creates users with the specified emails and assigns the selected role to them. 

Note: If a user already exists for a given email, the selected role will be assigned to them.
```

Reply:
Thanks, went with a slightly modified version:
```
Creates users with the specified emails and assigns the selected role to them.
If a user with a given email already exists, the selected role will be assigned to them.
```